docs/3rd-party-tools.md

```diff
@@ -118,14 +122,26 @@ sudo journalctl -u cross-seed # view the logs
 
 The way this works is you create a filter with a higher priority set than any other filter to make sure every cross-seed match is forwarded to the cross-seed daemon instead of being run through other filters.
 
-1. Create a filter and name it eg. `cross-seed`.
-2. Select all the indexers you want to use, preferably all of them.
-3. Set a really high `priority` to make sure it's always higher than your other filters.
-4. Go to the `External` tab, enable the `Webhook` switch, and add the following below it:
+1. Get your API key with the following command:  
+  
+    ```
+    cross-seed api-key
+    ```
+    Keep this key at hand since we will need it at step 5 later on.  
+    In the rest of this tutorial, we will refer to this as `YOUR_API_KEY`.
 
-   Host: `http://localhost:2468/api/announce`  
-   Expected http status: `200`  
-   Data (JSON):
+
+2. Create a filter and name it eg. `cross-seed`.
+3. Select all the indexers you want to use, preferably all of them.
+4. Set a really high `priority` to make sure it's always higher than your other filters.
+5. Go to the `External` tab, enable the `Webhook` switch, and add the following below it:
+
+
+   - Host: `http://localhost:2468/api/announce?apikey=YOUR_API_KEY`  
```

Comment:
z made sure it accepts the `x-api-key` header, so maybe we should recommend that approach?

Reply:
Sure can you do the changes or do you want me to do them?
Note: https://github.com/cross-seed/cross-seed/pull/543 there is a bug with the URL param too currently

Reply:
> Sure can you do the changes or do you want me to do them? Note: [cross-seed/cross-seed#543](https://github.com/cross-seed/cross-seed/pull/543) there is a bug with the URL param too currently

I just merged the PR to master, should be pushed to :latest soon.